jwst/datamodels/schemas/core.schema.yaml

```diff
@@ -658,28 +686,28 @@ properties:
             fits_keyword: TSAMPLE
             blend_table: True
           frame_time:
-            title: Time between frames (sec)
+            title: "[s] Time between frames"
             type: number
             fits_keyword: TFRAME
             blend_table: True
           group_time:
-            title: Time between groups (sec)
+            title: "[s] Time between groups"     
```

Comment:
It looks as if there are trailing blanks at the end of the group_time title.  Was that intentional?